src/main/java/frc/robot/RobotContainer.java

```diff
@@ -49,6 +49,7 @@ public RobotContainer() {
 
     // shooter.setDefaultCommand(new ShooterTest(shooter));
     // intake.setDefaultCommand(new IntakeTest(intake));
+    shooter.setDefaultCommand(new ShooterTestCommand(shooter));
```

Comment:
We don't want this to be the default command...

Reply:
Ok, I just did that trying to test the command, it was having problems and I was trying to make sure the command was being called. I meant to get rid of it when I was done.